src/Selector/IsException.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace PHPat\Selector;
+
+use PHPStan\Reflection\ClassReflection;
+
+final class IsException implements SelectorInterface
+{
+    public function getName(): string
+    {
+        return '-all exceptions-';
+    }
+
+    public function matches(ClassReflection $classReflection): bool
+    {
+        return $classReflection->implementsInterface(\Throwable::class);
```

Comment:
All Exceptions are Throwable, but not all Throwables are Exceptions. Strictly, Throwable includes Errors. Exceptions are only the classes that extend \Exception.